apps/test/unit/util/imageUtilsTest.js

```diff
@@ -4,14 +4,21 @@ import {
   dataURIToFramedBlob
 } from '@cdo/apps/imageUtils';
 import {expect} from '../../util/deprecatedChai';
-import expected from './expected.png';
+import expectedPhantomPng from './expected-phantom.png';
+import expectedChromePng from './expected-chrome.png';
 
 describe('image utils', () => {
   it('overlays an image inside the Artist frame', done => {
+    // Chrome and PhantomJS generate PNGs that _look_ the same but are are very different when
+    // you examine the files.  For now, we have fixtures for both since we want these tests to
+    // pass in both browsers.
+    const expectedPng = /PhantomJS/.test(window.navigator.userAgent)
+      ? expectedPhantomPng
+      : expectedChromePng;
```

Comment:
Similar to [the problem with loadAppTest](https://github.com/code-dot-org/code-dot-org/pull/32522#discussion_r360608895), Chrome and Phantom generate visually similar images that are very different when you actually try to do a bytewise comparison.

| PhantomJS | ChromeHeadless |
| --- | --- |
| 523kB | 40kB |
| ![expected](https://user-images.githubusercontent.com/1615761/71299500-6656ce00-2342-11ea-9508-4fb7da0254dc.png) | ![expected-chrome](https://user-images.githubusercontent.com/1615761/71299503-6a82eb80-2342-11ea-9ecf-6f69a36555d9.png) |